### DISCUSSION TITLE
Add btrfs support

### DISCUSSION THREAD

**Discussion by elias-hatami** [ventoy/Ventoy#2174] [General]
hello
Add btrfs support, eg browse btrfs partition when browsing file (f2).
Thanks

**Comment by dnchshrp** [comment]
That would be really nice

**Comment by hexman1999** [comment]
I agree totally

**Comment by Anonymous941** [comment]
Even without compression support it would be great, because you can have a bootable flash drive that's also used for normal file storage

**Comment by ventoy** [comment]
Ventoy 1.1.09 has introduced experimental support for btrfs.
